### PR TITLE
Adds Feedback and Language Swapper

### LINKED ISSUE #163: Update draft components to match S6 MVP
### Description

Update the current component layout to match the latest (SP6) MVP

 - [Slack link to the figma](https://nava.slack.com/archives/C01Q5640YFR/p1620829642029900)

SP6 MVP | New/Changed components
--|--
Claim Tracker SP6 MVP | Changed or added components on MVP

### Acceptance Criteria

  - [x] Add new component Language Switcher
  - [x] Add new component Feedback Box
  - [x] Claim Card -> Claim Section
  - [x] Claim Card Body -> Claim Status
  - [x] Move Claim Card Details to bottom
  - [x] Update Claim Status and Card Details placeholder
  - [x] Roughly correct styling

### PR DESCRIPTION
**Pull request by lomky**
Resolves #163 

#179 
  - [x] Add new component Language Switcher
  - [x] Add new component Feedback Box

Stack 1/3: #178
 - Moves all the files to their new location
Stack 2/3:  #179 <-- THIS PR
 - Adds the new components
Stack 3/3: #180 
 - Updates the Claim components

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const parameters = {
   options: {
     storySort: {
       method: 'alphabetical',
-      order: ['Claim Tracker', 'Component', ['Atom', 'Page Section', ['Header', 'Main', 'Claim Card', 'Claim Card Details', 'Claim Card Body', 'Footer']], 'Example'],
+      order: ['Claim Tracker', 'Component', ['Page Section', ['Header', 'Main', 'Claim Section', 'Claim Status', 'Claim Details', 'Feedback', 'Footer']], 'Atoms'],
```
Comment (lomky) [inline .storybook/preview.js:24]: this is just reordering the storybook sidebar

File: components/Feedback.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+export const Feedback: React.FC = () => {
+  const { t } = useTranslation('common')
+
+  return (
+    <div className="feedback">
+      <div className="feedback-content">
+        <div className="feedback-title">{t('feedback.title')}</div>
+        <div className="feedback-message">
+          {
+            // TODO correct feedback link?
```
Comment (kevmo) [inline components/Feedback.tsx:13]: Would be good for this to be just a touch more specific, so another engineer could pick it up rapidly if need be, e.g. "Is this correct i18nKey attribute value?"  or "TODO: Insert correct href link." (took me a second to realize what this meant, for instance). 

Absolutely not a blocking comment, just a grumpy old programmer comment, someone who has written many similar TODOs. :)

**Review by kevmo — APPROVED** [review]
Looks great. 🚀

File: components/Feedback.tsx (hunk 2 above)
Comment (lomky) [reply components/Feedback.tsx:13]: honestly I owe a pass on finding my TODOs and confirming tickets exist - I'll do that now.

**Pull request merged**